lndb/_notebook.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+from lamin_logger import logger
+
+
+def track(
+    notebook_path: str, parent: Optional[str] = None, pypackage: Optional[str] = None
+):
+    try:
```

Comment:
nbproject is a requirement of lamindb; hence, this isn't needed - but of course it doesn't hurt much! :)

Reply:
Yes, but not of lndb. Just to avoid any strange edge cases.